src/main/java/fr/esgi/al5_2/Tayarim/services/ProprietaireService.java

```diff
@@ -77,11 +84,11 @@ public ProprietaireDTO getProprietaireByEmail(@NonNull String email){
     public ProprietaireDTO updateProprietaire(@NonNull Long id, @NonNull ProprietaireUpdateDTO proprietaireUpdateDTO){
 
         if(
-                proprietaireUpdateDTO.getNom() == null
-                        && proprietaireUpdateDTO.getPrenom() == null
-                        && proprietaireUpdateDTO.getEmail() == null
-                        && proprietaireUpdateDTO.getNumTel() == null
-                        && proprietaireUpdateDTO.getMotDePasse() == null
+                (proprietaireUpdateDTO.getNom() == null || proprietaireUpdateDTO.getNom().isBlank())
+                        && (proprietaireUpdateDTO.getPrenom() == null || proprietaireUpdateDTO.getPrenom().isBlank())
+                        && (proprietaireUpdateDTO.getEmail() == null || proprietaireUpdateDTO.getEmail().isBlank())
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line is longer than 100 characters (found 115).